libraries/PDM/src/PDM.h

```diff
@@ -38,13 +38,20 @@ class PDMClass {
 
     //PORTENTA_H7 min -12 max 51
     //NANO 33 BLE SENSe min 0 max 80
+    //NICLA_VISION min 0 max 8
     void setGain(int gain);
     void setBufferSize(int bufferSize);
     size_t getBufferSize();
 
     // private:
     void IrqHandler(bool halftranfer);
 
+    // Hardware peripherals used
```

Comment:
Why would these be made public?  Nobody but the class should mess with the PIO program, no?